### PR TITLE
Add Currency To Refund

### PR DESCRIPTION
**Pull request by MitchHamm**
Shopify will be requiring this with their new multi currency

### CODE REVIEW

File: src/Models/Refund.php [hunk 1]
```diff
@@ -151,6 +154,22 @@ public function getShipping()
...
+    }
+
+    /**
+     * @param string $currency
+     */
+    public function setCurrency($currency)
+    {
+        $this->currency = $currency;
+    }
+
```
Comment (bmansell) [inline src/Models/Refund.php]: This block should be above line 149 so that it doesn't break up the getter/setter for `shipping`.

Comment (MitchHamm) [reply src/Models/Refund.php]: Fixed

File: tests/RefundTest.php [hunk 2]
```diff
@@ -35,6 +35,7 @@ public function ShopifyRefundSerializesProperly()
         $refundLineItem->setRestockType('cancel');
         $refundLineItem->setLocationId(777777);
         $refundLineItem->setQuantity(1);
+        $refundEntity->setCurrency('USD');
```
Comment (bmansell) [inline tests/RefundTest.php]: This line should be grouped with the `$refundEntity` declarations above (after line 31).

Comment (MitchHamm) [reply tests/RefundTest.php]: Done

**Pull request merged**